EU4toV2/Source/V2World/Country/Country.cpp

```diff
@@ -706,6 +706,13 @@ void V2::Country::newCivConversionMethod(double topTech, int topInstitutions, co
 
 	details.literacy *= theConfiguration.getMaxLiteracy() * (pow(10, civLevel / 100 * 0.9 + 0.1) / 10);
 
+	// drop nominal literacy according to starting date.
+
+	const auto diffYear = std::fmax(theConfiguration.getLastEU4Date().diffInYears(date("0.1.1")), 0);
+	const auto yearFactor = (0.1 + 4614700 * diffYear) / (1 + static_cast<float>(103810000) * diffYear - 54029 * pow(diffYear, 2));
```

Comment:
Are the constants just chosen to make a curve that fulfills the properties given in the description? If so, please add to the comment explaining so and giving the critical points for the curve. If they have other significance, maybe use constexpr constants to clarify their significance.

Also, instead of ```static_cast<float>(103810000)```, you should just be able to do ```103810000.0f```

Finally, since these are longer numbers, use the apostrophe operator to break them into sections: ```4'614'700```

Reply:
Yeah, dummy numbers to hit given points. Explained now and fixed the remainder of comments.